less/components/sidebar.less

```diff
@@ -19,7 +19,7 @@ html.no-js .sidebar {
   &.teach-like-mozilla .sidebar {
     .sidebarColorizer(@tealSidebarColor);
   }
-  &.clubs .sidebar {
+  &.clubs .sidebar, &.clubs-curriculum-page .sidebar {
```

Comment:
Interesting, I worked on a sub page, and I was given an off colour version of the parent page's colour. Maybe we can request a darker orange for clubs-curriculum?